src/wallet/wallet.cpp

```diff
@@ -2114,6 +2114,44 @@ void CWallet::ResendWalletTransactions(int64_t nBestBlockTime, CConnman* connman
  * @{
  */
 
+// void CWalletTx::AddBalances(CAmount& balance, CAmount& unconfirmed_balance, CAmount& immature_balance, bool& have_watch_only, CAmount& watch_only_balance, CAmount& unconfirmed_watch_only_balance, CAmount& immature_watch_only_balance) const
```

Comment:
:eyes:

Reply:
Heh

Reply:
BTW this is another experiment where many computations are deduped per wallet tx. Still WIP.